docs/pages/database-access/guides/redis-aws.mdx

```diff
@@ -5,6 +5,22 @@ description: How to configure Teleport database access with AWS ElastiCache and
 
 (!docs/pages/includes/database-access/db-introduction.mdx  dbType="AWS ElastiCache and MemoryDB for Redis cluster" dbConfigure="AWS ElastiCache and MemoryDB for Redis cluster database with IAM authentication" dbName="AWS ElastiCache and MemoryDB for Redis" !)
 
+## How it works
+
+The Teleport Database Service proxies traffic from users to AWS ElastiCache or
+MemoryDB for Redis. Authentication between the Database Service and the
+AWS-hosted Redis database can take one of two forms:
+
+- **IAM authentication (preferred):** The Teleport Database Service connects to
+  the database using a short-lived AWS IAM authentication token. AWS IAM
+  authentication is available for ElastiCache and MemoryDB with Redis version
+  7.0 or above.
+- **Managing users:** The Teleport Database Service manages users in a Redis
+  access control list, rotates their passwords every 15 minutes, and saves these
+  passwords in AWS Secrets Manager. The Database Service automatically sends an
+  `AUTH` command with the saved password when connecting the client to the Redis
+  server.
```

Comment:
@greedy52 Think we can deprecate this method anytime soon?

Reply:
IAM auth is only supported for Redis 7.0 +, so it will take a while to deprecate the old method:
https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/supported-engine-versions.html